client/my-sites/checkout/composite-checkout/payment-methods/paypal.tsx

```diff
@@ -264,12 +279,24 @@ function PayPalSubmitButton( {
 	const { __ } = useI18n();
 	const { formStatus } = useFormStatus();
 	const { transactionStatus } = useTransactionStatus();
-	const postalCode = useSelect( ( select ) => select( storeKey )?.getPostalCode() );
-	const countryCode = useSelect( ( select ) => select( storeKey )?.getCountryCode() );
-	const address1 = useSelect( ( select ) => select( storeKey )?.getAddress1() );
-	const organization = useSelect( ( select ) => select( storeKey )?.getOrganization() );
-	const city = useSelect( ( select ) => select( storeKey ).getCity() );
-	const state = useSelect( ( select ) => select( storeKey ).getState() );
+	const postalCode = useSelect(
+		( select ) => ( select( storeKey ) as PaypalSelectors ).getPostalCode(),
+		[]
+	);
+	const countryCode = useSelect(
+		( select ) => ( select( storeKey ) as PaypalSelectors ).getCountryCode(),
+		[]
+	);
+	const address1 = useSelect(
+		( select ) => ( select( storeKey ) as PaypalSelectors ).getAddress1(),
+		[]
+	);
+	const organization = useSelect(
+		( select ) => ( select( storeKey ) as PaypalSelectors ).getOrganization(),
+		[]
+	);
+	const city = useSelect( ( select ) => ( select( storeKey ) as PaypalSelectors ).getCity(), [] );
+	const state = useSelect( ( select ) => ( select( storeKey ) as PaypalSelectors ).getState(), [] );
```

Comment:
Given the old code used `?.` to access the selectors, it might be possible for them to be undefined

Reply:
On second look, it looks like select(storeKey) doesn't ever return undefined here, so we don't need the same logic.